docs/api/task.md

```diff
@@ -46,4 +46,4 @@ Sample `task` resource
 
 * [Get all tasks _(coming soon)_](#resource-properties)
 * [Get task by ID _(coming soon)_](#resource-properties)
-* [Get task by user ID _(coming soon)_](#resource-properties)
+* [Get task by user ID](#task-get-task-by-user-id)
```

Comment:
Double check the link. With the `#`, it will look for a heading in the current page with that title.